docs/Explanation/ADR/adr-005-license-credential.md

```diff
@@ -0,0 +1,380 @@
+
+This ADR describes the structure of a Financial Services License based on the [W3C specification](https://www.w3.org/TR/vc-data-model/) for verifiable credential. This credential will be signed by the relevant authority and can be used as proof of identity (for example, when establishing relationships with new clients) and proof that holder can perform claimed services.
+
+With respect to Cosmos Cash, any legal person with a suitable license can present this evidence of authority and perform network services, such as: 
+
+* Mint or burn e-money tokens
+* Operate a Decentralized Exchange 
+* etc. 
+
+---
+
+## Context
+
+[ADR-003 defines an issuer module](https://github.com/allinbits/cosmos-cash/blob/main/docs/Explanation/ADR/adr-003-issuer.md). One of the actors defined in that ADR was an issuer. This issuer would hold a license that gives permissions to perform certain activities. The license is issued by a authority actor.
+
+The use cases of Role-based permissions on-chain extends to other services or protocols. For example, exchange and custody services. A definition that can extend to these cases would be vital.
+
+Current regulation is typically based around a single jurisdiction.  Each jurisdiction can have one or more authorities. Each authority manages one or more financial services and licenses legal persons for these services. In effect a single legal person can have one or more licenses to provide different services from different authorities. 
+
+In the case of the EU, regulation can extend across multiple jurisdiction, specifically passporting means that licensed legal persons in one jurisdiction can offer the same financial services in another EU-based jurisdiction. 
+
+[OpenVASP](https://openvasp.org) defines a [credential](https://github.com/OpenVASP/ovips/blob/master/ovip-0015.md) and [ovip-0013](https://github.com/OpenVASP/ovips/blob/master/ovip-0013.md) as part of their protocol. 
+
+### Terminology
+
+* `Legal Person`: any person or entity that is able to perform legal activties, such as enter into contracts, own property, and so on
+* `Virtual Asset Service Provider`: FATF defines a VASP as the following: *“Any natural/legal person who ...  as a business conducts one or more of the following activities or operations for or on behalf of another natural or legal person: i. exchange between virtual assets and fiat currencies; ii. exchange between one or more forms of virtual assets; iii. transfer of virtual assets; iv. safekeeping and/or administration of virtual assets or instruments enabling control over virtual assets; and v. participation in and provision of financial services related to an issuer’s offer and/or sale of a virtual asset.”*
```

Comment:
source needed?

Reply:
Yes, I can add this.

Reply:
This section has been added to the glossary and a citation added for VASP definition.